.github/workflows/main.yml

```diff
@@ -0,0 +1,96 @@
+name: CI
```

Comment:
Maybe, if possible, we need to make this file a little more consistent with our other files:
https://github.com/skedify/sdk-js/blob/develop/.github/workflows/ci.yml